<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uploadWindow</class>
 <widget class="QMainWindow" name="uploadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Upload New Image</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myApp.qrc">
    <normaloff>:/APPICONS/icons/hd-icon.png</normaloff>:/APPICONS/icons/hd-icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(64, 105, 159);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>831</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="originalImageLabel">
       <property name="styleSheet">
        <string notr="true">color: rgb(144, 170, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Original Image:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="OriginalImage_lineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BrowseButton_Original">
       <property name="styleSheet">
        <string notr="true">color: rgb(144, 170, 255);</string>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="UploadButton_Original">
       <property name="styleSheet">
        <string notr="true">color: rgb(144, 170, 255);</string>
       </property>
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>831</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="fakeImageLabel">
       <property name="styleSheet">
        <string notr="true">color: rgb(144, 170, 255);</string>
       </property>
       <property name="text">
        <string>    Fake Image:  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fakeImage_lineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BrowseButton_fake">
       <property name="styleSheet">
        <string notr="true">color: rgb(144, 170, 255);</string>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="UploadButton_fake">
       <property name="styleSheet">
        <string notr="true">color: rgb(144, 170, 255);</string>
       </property>
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(144, 170, 255);</string>
    </property>
    <property name="text">
     <string>    Original Image  </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>80</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(144, 170, 255);</string>
    </property>
    <property name="text">
     <string>    Fake Image  </string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>85</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(144, 170, 255);</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>520</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>520</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Orignial_viewer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>121</y>
      <width>930</width>
      <height>540</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>938</x>
      <y>121</y>
      <width>20</width>
      <height>531</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Fake_viewer">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>121</y>
      <width>930</width>
      <height>540</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="myApp.qrc"/>
 </resources>
 <connections/>
</ui>
